todo_app/templates/index.html

```diff
@@ -1,78 +1,149 @@
                     border-collapse: collapse;
+                    border-spacing: 0;
                     padding: 5px;
+                    border-radius: 4px;
                 }
                 
-                .demo th {
+                /* .demo th {
                     border: 1px solid #C0C0C0;
                     padding: 5px;
                     background: #F0F0F0;
-                }
-                
+                } */
+/*                 
                 .demo td {
                     border: 1px solid #C0C0C0;
                     padding: 5px;
-                }
-            </style>
-            <table class="demo">
-                <caption>Add A Task</caption>
-               
-                <thead>
+                } */
+                .demo th, td {
+                   text-align: left;
+                    padding: 16px;
+                    border: 1px solid #C0C0C0;
+                    border-radius: 4px;
+                   
+}
+.form{ width: 32%;
+  padding: 12px 20px;
+  margin: 8px 0;
+  box-sizing: border-box;
+}
+input[type=text], select {
+  width: 32%;
+  padding: 5px;
+  margin: 8px 0;
+  display: inline-block;
+  border: 1px solid #ccc;
+  border-radius: 4px;
+  box-sizing: border-box;
+}
+
+input[type=submit] {
+  width: 20%;
+  background-color: #4CAF50;
+  color: white;
+  padding: 5px;
+  margin: 8px 0;
+  border: none;
+  border-radius: 4px;
+  cursor: pointer;
+}
+
+input[type=submit]:hover {
+  background-color:#4CAF50 ;
+}
+
+
+
+.container{min-width: fit-content}/* Responsive layout - makes the two columns stack on top of each other instead of next to each other on screens that are smaller than 600 px */
+@media screen and (min-width: fit-content) {
+  .column {
+    width: 100%;
+ 
+  }
+}
+    </style>
+</head>
+<body>
+    <h2>Trying To Make It Like Jira Project</h2>
+        <div class="row">
+            <div class="column">
+            <table >
+
                     <tr>
                         <th>Agenda</th>
                         <th>Status</th>
                         <th>Doing</th>
                         <th>Done</th>
-                    
+                        <th>Delete</th>
                     </tr>
                     {% for item in to_do %}
-                </thead>
-                <tbody>
                     <tr>
                         <td>{{item.title}}</td>
-                        <td>{{item.status}}</td>
-                    
-                        <td>{{item.status}}</td>
-                        <td>{{item.status}}</td>
-                        
+                        <td>{{item.status}};</td>
+                        <td>
+                            <form action="/doingitem " method="post">
+                                <input hidden type=text name="doing_item_form" value="{{item.id}}">
+                                <button type="submit " name="inprogress ">start</button>
+                            </form>
```

Comment:
You've got some extra spaces in here

```suggestion
                            <form action="/doingitem" method="post">
                                <input hidden type=text name="doing_item_form" value="{{item.id}}">
                                <button type="submit" name="inprogress">start</button>
                            </form>
```